Cognite.Simulator.Tests/SeedData.cs

```diff
@@ -245,38 +300,31 @@ public static async Task<SimulatorRoutineRevision> GetOrCreateSimulatorRoutineRe
                 Schedule = new SimulatorRoutineRevisionSchedule()
                 {
                     Enabled = true,
-                    StartTime = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds() - 10000,
-                    Repeat = "5s",
+                    CronExpression = "*/5 * * * *",
                 },
                 DataSampling = new SimulatorRoutineRevisionDataSampling()
                 {
+                    Enabled = true,
                     ValidationWindow = 1440,
                     SamplingWindow = 60,
                     Granularity = 1,
-                    ValidationEndOffset = "10m"
-                },
-                LogicalCheck = new SimulatorRoutineRevisionLogicalCheck()
-                {
-                    Enabled = false,
-                },
-                SteadyStateDetection = new SimulatorRoutineRevisionSteadyStateDetection()
-                {
-                    Enabled = false,
                 },
+                LogicalCheck = new List<SimulatorRoutineRevisionLogicalCheck>(),
+                SteadyStateDetection = new List<SimulatorRoutineRevisionSteadyStateDetection>(),
                 InputConstants = new List<SimulatorRoutineRevisionInputConstants>(),
                 InputTimeseries = new List<SimulatorRoutineRevisionInputTimeseries>(),
                 OutputTimeseries = new List<SimulatorRoutineRevisionOutputTimeseries>(),
             },
-            ExternalId = "Test Scheduled Routine - 2",
+            ExternalId = "Test Scheduled Routine - 3",
```

Comment:
```suggestion
            ExternalId = "Test Scheduled Routine - 2",
```
there are only two of them, this is less confusing